osu.Game/Localisation/CommonStrings.cs

```diff
@@ -179,6 +179,16 @@ public static class CommonStrings
         /// </summary>
         public static LocalisableString CopyLink => new TranslatableString(getKey(@"copy_link"), @"Copy link");
 
+        /// <summary>
+        /// "4K"
+        /// </summary>
+        public static LocalisableString FourKey => new TranslatableString(getKey(@"four_key"), @"4K");
+
+        /// <summary>
+        /// "7K"
```

Comment:
There's already strings in web:
https://github.com/ppy/osu-resources/blob/5642a9c5f18b5c17a2f7508f12f295524b976a71/osu.Game.Resources/Localisation/Web/BeatmapsStrings.cs#L1207-L1215